bukkit/src/main/java/net/william278/huskhomes/util/BukkitSafetyUtil.java

```diff
@@ -48,6 +48,7 @@ public static Optional<Location> findSafeLocation(@NotNull World world, @NotNull
      * @return True if the block is safe, false otherwise
      */
     public static boolean isSafePosition(@NotNull ChunkSnapshot chunkSnapshot, final int chunkX, final int chunkY, final int chunkZ) {
+        if(chunkY<-64||chunkY>320)return false;
```

Comment:
This will throw exceptions on 1.16 servers, still, as those have a different world height -- can we use org.bukkit.World#getWorldHeight and the world min height methods?

Reply:
you're right. i didn't think about past versions. i'll fix it asap